plans/full.fmf

```diff
@@ -0,0 +1,10 @@
+summary:
+  Unit & integration tests.
+discover+:
+  filter: tier:1
+prepare:
```

Comment:
I'd suggest using tmt's `install` method to avoid calling `dnf` directly. It handles other things such as running sudo if necessary and should also work on silverblue:

```yaml
prepare:
  - name: pip
    how: install
    package: python3-pip
  - name: flexmock
    how: shell
    # flexmock is not in EPEL 9, install it from PyPI
    script: pip3 install flexmock
```

Reply:
Thanks, this is much better.